eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -266,6 +266,20 @@ jobs:
     displayName: Prepare BuildLogs staging directory
     continueOnError: true
     condition: succeededOrFailed()
+
+  - ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
```

Comment:
Do we want to limit this to Pull Requests? What if someone runs a manual build in public, shouldn't it upload the logs?

Especially given that the `Build` job does not have the same conditional

Reply:
Should this be triggered on `eq(variables['System.TeamProject'], 'public') }}:` instead?  Do you get the right behavior for internal PRs?